templates/field/timeline.html.twig

```diff
@@ -0,0 +1,13 @@
+{#
+/**
+ * @file
+ * Theme override for the timeline field template.
+ *
+ * @see ./contrib/oe_content/oe_content_timeline/templates/timeline.html.twig
```

Comment:
I'd remove this `see` thing. The path is also wrong and it assumes our own project structure.